meta-mender-core/recipes-bsp/u-boot/patches/0002-Generic-boot-code-for-Mender.patch

```diff
@@ -166,7 +166,13 @@ index 0000000..705c39e
 +                                                                        \
 +    "mender_dtb_name=" MENDER_DTB_NAME "\0"                             \
 +                                                                        \
++    "mender_pre_setup_commands=" MENDER_PRE_SETUP_COMMANDS "\0"         \
++                                                                        \
 +    "mender_setup="                                                     \
++    "if test \"${mender_pre_setup_commands}\" != \"\"; "                \
++    "then "                                                             \
++    "run mender_pre_setup_commands; "                                   \
++    "fi; "                                                              \
```

Comment:
If following the suggestion above, this should be the only remaining piece of this patch.